pyproject.toml

```diff
@@ -92,15 +94,29 @@ log_cli_level = "INFO"
 xfail_strict = true
 filterwarnings = [
     "error",
-    "ignore:.*pkg_resources.*:DeprecationWarning:",
-    "ignore:.*sre_.*:DeprecationWarning:",
-    "ignore:.*Rigetti.*:UserWarning:",
-    "ignore:.*Values in x.*:RuntimeWarning:",
-    "ignore:.*divide by zero encountered in det.*:RuntimeWarning:",
-    "ignore:.*invalid value encountered in det.*:RuntimeWarning:",
-    "ignore::DeprecationWarning:.*(docplex).*",
-    "ignore::SyntaxWarning:.*(docplex).*",
-    "ignore:.*The abstract Provider and ProviderV1 classes are deprecated.*:DeprecationWarning:qiskit.*",
+    'ignore:.*pkg_resources.*:DeprecationWarning:',
+    'ignore:.*sre_.*:DeprecationWarning:',
+    'ignore:.*Rigetti.*:UserWarning:',
+    'ignore:.*Values in x.*:RuntimeWarning:',
+    'ignore:.*divide by zero encountered in det.*:RuntimeWarning:',
+    'ignore:.*invalid value encountered in det.*:RuntimeWarning:',
+    'ignore::DeprecationWarning:.*(docplex).*',
+    'ignore::SyntaxWarning:.*(docplex).*',
+    'ignore:.*The abstract Provider and ProviderV1 classes are deprecated.*:DeprecationWarning:qiskit.*',
+    'ignore:.*qiskit.providers.models is deprecated since Qiskit 1.2*:DeprecationWarning:',
+    'ignore:.*Treating CircuitInstruction as an iterable is deprecated legacy behavior since Qiskit 1.2*:DeprecationWarning:pytket.*',
+    'ignore:.*The class ``qiskit.providers.models.backendconfiguration.GateConfig`` is deprecated as of qiskit 1.2.*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.providers.models.backendconfiguration.BackendConfiguration`` is deprecated as of qiskit 1.2.*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.providers.models.backendconfiguration.QasmBackendConfiguration`` is deprecated as of qiskit 1.2.*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.providers.models.backendproperties.BackendProperties`` is deprecated as of qiskit 1.2.*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.qobj.pulse_qobj.PulseLibraryItem`` is deprecated as of qiskit 1.2.*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.primitives.base.base_estimator.BaseEstimator`` is deprecated as of qiskit 1.2*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.primitives.base.base_sampler.BaseSampler`` is deprecated as of qiskit 1.2*:DeprecationWarning:qiskit.*',
+    'ignore:.*The class ``qiskit.primitives.sampler.Sampler`` is deprecated as of qiskit 1.2*:DeprecationWarning:mqt.*',
+    'ignore:.*The class ``qiskit.primitives.estimator.Estimator`` is deprecated as of qiskit 1.2*:DeprecationWarning:mqt.*',
```

Comment:
If those are coming from our own library, then we should fix them. There are direct replacements for the sampler and the estimator.